modules/Makefile.kubernetes

```diff
@@ -26,6 +26,9 @@ endef
 define kubectl_apply
 	$(call assert,CLUSTER_NAMESPACE)
 	$(call assert,CLUSTER_DOMAIN)
+	# Optionally load parameters for the kubernetes resources
+	-@source $(KUBERNETES_RESOURCE_PATH)/default.k8sparameters
+	-@source $(KUBERNETES_RESOURCE_PATH)/$(CLUSTER_NAMESPACE).$(CLUSTER_DOMAIN).k8sparameters
 	@echo -e "INFO: Applying changes to $(KUBERNETES_APP) $(1) on cluster $(call yellow,$(CLUSTER))..."
 	@$(call envsubst,$(KUBERNETES_RESOURCE_PATH)/$(KUBERNETES_APP)-$(1).yml) | $(KUBECTL_CMD) apply $(2) -f -
```

Comment:
this is the line that will make env variables work, we already set $IMAGE_TAG through this mechanism